src/app/components/password-requirement/password-requirement.component.scss

```diff
@@ -1,89 +1,86 @@
-  gap: 8px;
-  list-style: none;
-  padding: 0;
-  margin: 0;
-}
 
-.list__requirement mat-icon {
-  font-size: 20px;
-  height: 20px;
-}
-
-.list__requirement li {
-  display: flex;
-  align-items: center;
-  gap: 8px;
-}
+  &__item-password {
+    &__form-item {
+      width: 100%;
+    }
 
-.list__requirement span {
-  font-family: Roboto;
-  font-size: 14px;
-  font-style: normal;
-  font-weight: 500;
-  line-height: normal;
-  letter-spacing: -0.07px;
-}
-
-.section__item__requirement {
-  display: flex;
-  padding: 16px;
-  flex-direction: column;
-  align-items: flex-start;
-  gap: 10px;
-  border-radius: 4px;
-  border: 1px solid #373744;
-  margin-bottom: 22px;
-
-  mat-label {
-    color: #6B6B83;
-    cursor: url(../../../assets/icons/mouse-hover.png), auto;
+    &__form-item__eyes {
+      color: #FFFFFF;
+      cursor: url("../../../assets/icons/mini-pointer-2.png"), auto;
+    }
   }
-}
-
-::ng-deep.mdc-text-field--filled:not(.mdc-text-field--disabled) .mdc-text-field__input {
-  cursor: url("../../../assets/icons/mouse-hover.png"), auto;
-}
 
-::ng-deep.mdc-text-field--filled:not(.mdc-text-field--disabled) {
-  cursor: url("../../../assets/icons/mouse-hover.png"), auto;
-}
+  &__item-requirement {
+    display: flex;
+    padding: 16px;
+    flex-direction: column;
+    align-items: flex-start;
+    gap: 10px;
+    border-radius: 4px;
+    border: 1px solid #373744;
+    margin-bottom: 22px;
+
+    mat-label {
+      color: #6B6B83;
+      cursor: url(../../../assets/icons/mouse-hover.png), auto;
+    }
+
+    &__list-requirement {
+      display: flex;
+      flex-direction: column;
+      gap: 8px;
+      list-style: none;
+      padding: 0;
+      margin: 0;
+
+      mat-icon {
+        font-size: 20px;
+        height: 20px;
+      }
+
+      li {
+        display: flex;
+        align-items: center;
+        gap: 8px;
+      }
+
+      span {
+        font-family: Roboto;
+        font-size: 14px;
+        font-style: normal;
+        font-weight: 500;
+        line-height: normal;
+        letter-spacing: -0.07px;
+      }
+
+      .valid {
+        color: $success-text-color;
+      }
+
+      .invalid {
+        color: $error-text-color;
+      }
+    }
+  }
 
-::ng-deep .mat-mdc-button .mat-mdc-button-touch-target, .mat-mdc-unelevated-button .mat-mdc-button-touch-target, .mat-mdc-raised-button .mat-mdc-button-touch-target, .mat-mdc-outlined-button .mat-mdc-button-touch-target {
-  cursor: url(../../../assets/icons/mini-pointer-2.png), auto;
-}
+  &__confirmPassword {
+    text-align: left;
 
-::ng-deep .mat-mdc-form-field:not(.mat-form-field-disabled) .mat-mdc-floating-label.mdc-floating-label {
-  cursor: url("../../../assets/icons/mouse-hover.png"), auto;
-}
+    &__form-item {
+      width: 100%;
 
-.section__confirmPassword {
-  text-align: left;
-}
-
-.form__icon__eyes {
-  color: #FFFFFF;
-  cursor: url("../../../assets/icons/mini-pointer-2.png"), auto;
+      &__eyes {
+        color: #FFFFFF;
```

Comment:
Seria bom dar uma revisada nas cores, se possível pegar essas que estão soltas, colocar lá no arquivo de cores e reutilizar nos componentes.

Reply:
Gente, no root do scss de styles, tinham as variáveis com as cores chapadas e as váriaves não pegam as cores que a gente determinou na paleta de jeito nenhum. Isso, pelo que entendi inspecionando, a variável tá dentro de outra classe. Me parece um problema de hierarquia que nem ng-deep resolveu. .Então eu vi que se ao invés da variável eu mudar a cor na classe, dá pra usar nosso scss. Então tô trocando as varíaveis tudo por classe pra padronizar o uso de cores como  pedido na correção do PR.

Reply:
Tenho dúvidas sobre isso (as vezes eu acho que a gente tá reatribuindo muitas vezes pra messma cor):

no colors.scss:

// Cores
$accent-color: #6200EE;
$success-text-color: #17C69C;
$error-text-color: #EE0056;
$text-color: #fff;
$background-color: #25252F;
$secondary-background-color: #373744;
$input-color: #ffffff14;
$input-text-color: rgba(255, 255, 255, 0.87);
$active-button-color: #767676;

:root {
  --accent-color: $accent-color;
  --success-text-color: $success-text-color;
  --error-text-color: $error-text-color;
  --text-color: $text-color;
  --background-color: $background-color;
  --secondary-background-color: $secondary-background-color;
}

se já tem as cores do SCSS, é preciso criar variáveis também? 

outra dúvida: como falei acima, tinham variaveis do proprio angular material no root do style.scss. Qual a vantagem, uma vez que elas estão já sendo usadas em seus proprios componentes? Fiquei em dúvida, uma vez que já temos scss separados para espaçamento, cores, fontes, temas etc.

aqui está o root:

:root {

  .mdc-text-field--filled:not(.mdc-text-field--disabled) {
    background-color: $input-color;
  }

  .mat-mdc-form-field:not(.mat-form-field-no-animations) .mdc-text-field__input {
    color: $input-text-color;
  }

  .mat-mdc-form-field:not(.mat-form-field-disabled) .mat-mdc-floating-label.mdc-floating-label {
    color: $input-text-color !important;
  }
  .mat-mdc-input-element::placeholder {
       color: $input-text-color !important;
  }

  .mdc-text-field--filled:not(.mdc-text-field--disabled) .mdc-line-ripple::before {
    border-bottom-color: $accent-color;
  }
}

Reply:
e uma ultima dúvida aqui, as cores importadas não vão pegar no html do email né?